python/tskit/trees.py

```diff
@@ -574,6 +574,35 @@ def num_alleles(self) -> int:
         """
         return len(self.alleles) - self.has_missing_data
 
+    def concordance(self, other):
+        """
+        Calculate the fraction of samples for which the genotypes
+        of this variant are equal to the genotypes of the other variant.
+        The result is valid only if the genotypes for both the variants
+        are in the same order with respect to their samples.
+        Also, genotypes can be missing. Two genotypes with the value
```

Comment:
We're not really dealing with missing data here I think? We're saying missing data is equal to itself, but not equal to any alleles.

I think that's all we can do, but we should document it a bit more clearly.